poem-review-form.html

```diff
@@ -0,0 +1,185 @@
+				<li><a href="#">One more separated link</a></li>
+			  </ul>
+			</li> -->
+		  </ul>	
+          <ul class="nav navbar-nav navbar-right">
+        		<li><a href="#"><span class="glyphicon glyphicon-user" aria-hidden="true"></span> Sign In</a></li>
+      	  </ul>  
+        </div><!--/.navbar-collapse -->
+      </div>
+    </nav>
+
+    <!-- Main jumbotron for a primary marketing message or call to action -->
+    <div class="jumbotron">
+      <div class="container poem-pg-poem-title">
+        <h1>Leave a Review</h1>
+          <!-- replace the poem title and author with template-binding variables -->
+        <p class="review-form-poem-title">The Road Not Taken<br>
+            &nbsp&nbsp <span  class="review-form-poem-author">- by Robert Frost</span>
+        </p>
+      </div>
+    </div>
+
+    <div class="container container-home">
+      <!-- Example row of columns -->
+      <div class="row">
+
+
+          <div class="container poem-review-form">
+              <div class="row">
+                  <div class="col-xs-12 col-md-8">
+                      <!-- replace the action to return to poem page template-binding variables -->
+                      <form action="./poem.html" method="get" role="form" class="form-horizontal">
+                          <div class="form-group">
+                              <label for="name" class="col-xs-10 col-sm-2 control-label">Name</label>
+                              <div class="col-xs-12 col-sm-10">
+                                  <input id="name" name="name" class="form-control">
+                              </div>
+                          </div>
+                          <div class="form-group">
+                              <label for="rating" class="col-xs-10 col-sm-2 control-label">Rating</label>
+                              <div class="col-xs-12 col-sm-2">
+                                  <select id="rating" name="rating" class="form-control input-sm">
+                                      <option>5</option>
+                                      <option>4</option>
+                                      <option>3</option>
+                                      <option>2</option>
+                                      <option>1</option>
+                                  </select>
+                              </div>
+                          </div>
+                          <div class="form-group">
+                              <label for="review" class="col-sm-2 control-label">Review</label>
+                              <div class="col-sm-10">
+                                  <textarea id="review" name="review" rows="5" class="form-control"></textarea>
+                              </div>
+                          </div>
+                          <button class="btn btn-default pull-right">Add my review</button>
+                      </form>
+                  </div>
+              </div>
+          </div>
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+ 
```

Comment:
We should remove the empty lines from 127 - 150 and have the closing div and proceeding code on line 151 moved up to line 127.